charts/tigera-operator/values.yaml

```
@@ -5,18 +5,79 @@
 # Example: --set-file imagePullSecrets.gcr=./pull-secret.json
 imagePullSecrets: {}
 
+# Configures general installation parameters for Calico. Schema is based
+# on the operator.tigera.io/Installation API documented
+# here: https://docs.tigera.io/calico/latest/reference/installation/api#installationspec
 installation:
   enabled: true
-  kubernetesProvider: ""
+
```

```suggestion
  # Uncomment to enable prometheus metrics reporting for node and typha. 
  # nodeMetricsPort: 9090
  # typhaMetricsPort: 9091
```

^ This will address https://github.com/projectcalico/calico/issues/6887